catalogs/sources/gtfs/realtime/unknown-unknown-zarzad-transportu-publicznego-w-krakowie-ztp-krakow-gtfs-rt-vp-2340.json

```diff
@@ -0,0 +1,12 @@
+{
+    "mdb_source_id": 2340,
+    "data_type": "gtfs-rt",
+    "entity_type": [
+        "vp"
+    ],
+    "provider": "Zarząd Transportu Publicznego w Krakowie (ZTP Kraków)",
```

Comment:
@AlfredNwolisa Please add the static references to the realtime feeds (you can do this through the json files in github). They'll match the T or A value in the URL